### PR TITLE
Created the Judging Tutorial

### PR DESCRIPTION
**Pull request by denthebro**
This is a WQAC project that adds an edudoc that teaches newcomers the basics of judging WCA competitions.
This should be pulled at the same time as the corresponding wca website PR that adds the link to this file in the edudoc section.

### CODE REVIEW

**Comment by c-goodyear** [comment]
Board approved to be merged. @thewca/software-team please merge this when you can.

File: edudoc/judge-tutorial/judge-tutorial.md
Comment (gregorbg) [inline edudoc/judge-tutorial/judge-tutorial.md:33]: you shouly *always* ask the WCA Delegate

File: edudoc/judge-tutorial/judge-tutorial.md
Comment (gregorbg) [inline edudoc/judge-tutorial/judge-tutorial.md:39]: - starts the *solve*?
- before 17 seconds *of inspection time*?

File: edudoc/judge-tutorial/judge-tutorial.md
Comment (gregorbg) [inline edudoc/judge-tutorial/judge-tutorial.md:41]: - start the *solve*, see above

File: edudoc/judge-tutorial/judge-tutorial.md
Comment (gregorbg) [inline edudoc/judge-tutorial/judge-tutorial.md:25]: In roughly 8 years of competition experience, I have only noticed two occasions where judges call out the result loud.
1. If you're judging in a World Championship final where you're really adhering to the Regulations by the letter
2. If you're judging and your name happens to be Ron van Bruchem

File: edudoc/judge-tutorial/judge-tutorial.md
Comment (gregorbg) [inline edudoc/judge-tutorial/judge-tutorial.md:35]: Point out that +2 penalties can be accumulated? (Not like +12 extreme examples, but +4 *does* happen every now and then)

File: edudoc/judge-tutorial/judge-tutorial.md
Comment (gregorbg) [inline edudoc/judge-tutorial/judge-tutorial.md:72]: Is this in sync with the competitor guide? IIRC it also has an image/section about misalignments.

**Review by gregorbg — COMMENTED** [review]
Some general remarks but only regarding the content. They can be narrowed down to "personal taste" in most places, only the remark about +2 penalties being cumulative should be seriously considered imho.

Hope I'm not stepping on anyone's toes by making suggestions to the content... The Board or WQAC may override my concerns and tell us to merge this at any point in time; from a technical standpoint you have my LGTM!

File: edudoc/judge-tutorial/judge-tutorial.md
Comment (denthebro) [reply edudoc/judge-tutorial/judge-tutorial.md:39]: I changed it to "solve", but I feel like repeating "inspection time" again is a little clunky

File: edudoc/judge-tutorial/judge-tutorial.md
Comment (denthebro) [reply edudoc/judge-tutorial/judge-tutorial.md:25]: I strongly believe that we should still tell the judges to judge as stated in the regulations. Just because people don't do it, doesn't mean they shouldn't

File: edudoc/judge-tutorial/judge-tutorial.md
Comment (denthebro) [reply edudoc/judge-tutorial/judge-tutorial.md:72]: No, it actually isn't, but the competitor tutorial is super old and we have considered making a newer, more concise version in the foreseeable future

**Comment by denthebro** [comment]
@gregorbg made a new commit with most of your requested changes. If this looks okay, please merge along with website PR https://github.com/thewca/worldcubeassociation.org/pull/6302

File: edudoc/judge-tutorial/judge-tutorial.md
Comment (gregorbg) [reply edudoc/judge-tutorial/judge-tutorial.md:72]: Okay

File: edudoc/judge-tutorial/judge-tutorial.md
Comment (gregorbg) [reply edudoc/judge-tutorial/judge-tutorial.md:25]: Okay

**Review by gregorbg — APPROVED** [review]
Thanks, LGTM

**Pull request merged**
